ElDeveloper/metagenomics_pooling_notebook
# Amplicon pooling

Steps to test this is working:

- Clone the repo.
- Pull this branch.
- Install dependencies following README.md instructions.
- Open amplicon-pooling.ipynb
- Run the code and verify the outputs are there as expected.
